test/internal/constants/images.go

```diff
@@ -51,23 +72,24 @@ func makeImageURL(repo, image, tag string) string {
 }
 
 func NanoserverImage(tag string) string {
-	return makeImageURL(McrWindowsImageRepo, "nanoserver", tag)
+	return makeImageURL(MCRWindowsImageRepo, "nanoserver", tag)
 }
 
 func ServercoreImage(tag string) string {
-	return makeImageURL(McrWindowsImageRepo, "servercore", tag)
+	return makeImageURL(MCRWindowsImageRepo, "servercore", tag)
 }
 
 var _buildToTag = map[uint16]string{
-	osversion.RS1:         "1607",
-	osversion.RS2:         "1703",
-	osversion.RS3:         "1709",
-	osversion.RS4:         "1803",
-	osversion.RS5:         "1809",
-	osversion.V19H1:       "1903",
-	osversion.V19H2:       "1909",
-	osversion.V20H1:       "2004",
-	osversion.V21H2Server: "ltsc2022",
+	osversion.RS1:      "1607",
+	osversion.RS2:      "1703",
+	osversion.RS3:      "1709",
+	osversion.RS4:      "1803",
+	osversion.RS5:      "1809",
+	osversion.V19H1:    "1903",
+	osversion.V19H2:    "1909",
+	osversion.V20H1:    "2004",
+	osversion.V20H2:    "20H2",
+	osversion.LTSC2022: "ltsc2022",
```

Comment:
Should we also include the other new versions you added?

Reply:
I dont think, since there aren't image tags cut for that

Reply:
I meant like the "V1903" versions